src/mixins/onThisPageRegistrator.js

```diff
@@ -37,12 +37,18 @@ export default {
       this.store.resetPageSections();
       // register new ones
       const {
+        metadata: { title },
         primaryContentSections,
         topicSections,
         defaultImplementationsSections,
         relationshipsSections,
         seeAlsoSections,
       } = topicData;
+      this.store.addOnThisPageSection({
+        title,
+        anchor: 'app',
```

Comment:
Yep, that was my other thought too. Will check if it causes strange issues and do it.

Reply:
Added a dedicated component, because the scroll tracking logic was throwing a fuss for not being able to find a component to verify if on page or not.